newdle/schemas.py

```diff
@@ -8,3 +8,14 @@ class UserSchema(mm.Schema):
     name = fields.Function(lambda u: f'{u["first_name"]} {u["last_name"]}')
     initials = fields.Function(lambda u: f'{u["first_name"][0]} {u["last_name"][0]}')
     uid = fields.String()
+
+
+class UserSearchResultSchema(UserSchema):
+    class Meta:
+        fields = ('email', 'name', 'initials')
```

Comment:
Since you added this here, maybe you can use that in the `UserMenu` component?

Reply:
982b14c ;)

Reply:
Ohh, didn't notice that one :P